modules/op-release-notes-1-9.adoc

```diff
@@ -236,3 +236,119 @@ If you do not add any configuration data, you can use the default data in the AP
+
+[id="fixed-issues-1-9_{context}"]
+== Fixed issues
+
+// .Pipelines
+
+* Before this update, after replacing an empty array, the original array returned an empty string rendering the paramaters inside it invalid. With this update, this issue is resolved and the original array returns as empty. 
+// Vincent Demeester
+
+* Before this update, if duplicate secrets were present in a service account for a pipelines run, it resulted in failure in task pod creation. With this update, this issue is resolved and the task pod is created successfully even if duplicate secrets are present in a service account.
+// Vincent Demeester
+
+* Before this update, by looking at the TaskRun's `spec.StatusMessage` field, users could not distinguish whether the `TaskRun` had been cancelled by the user or by a `PipelineRun` that was part of it. With this update, this issue is resolved and users can distinguish the status of the `TaskRun` by looking at the TaskRun's `spec.StatusMessage` field.
+// Vincent Demeester
+
+* Before this update, webhook validation was removed on deletion of old versions of invalid objects. With this update, this issue is resolved.
+// Vincent Demeester @vdeemester
+
+* Before this update, a race condition could occur if another tool updated labels or annotations on a PipelineRun or TaskRun. With this update, this issue is resolved and you can merge labels or annotations.
+// Vincent Demeester @vdeemester
+
+// .Triggers
+
+* Before this update, log keys did not have the same keys as in pipelines controllers. With this update, this issue has been resolved and the log keys have been updated to match the log stream of pipeline controllers. The keys in logs have been changed from "ts" to "timestamp", from "level" to "severity", and from "message" to "msg".
+// https://issues.redhat.com/browse/SRVKP-1959
+// @KhurramBaig
+
+
+// .CLI
+
+* Before this update, if a PipelineRun was deleted with an unknown status, an error message was not generated. With this update, this issue is resolved and an error message is generated.
+// https://github.com/tektoncd/cli/pull/1684
+// Piyush Garg @piyush-garg
+
+* Before this update, to access bundle commands like `list` and `push`, it was required to use the `kubeconfig` file . With this update, this issue has been resolved and the `kubeconfig` file is not required to access bundle commands.
+// https://github.com/tektoncd/cli/pull/1557
+// Piyush Garg @piyush-garg
+
+* Before this update, if the parent pipelinerun was running while deleting TaskRuns, then TaskRuns would be deleted. With this update, this issue is resolved and TaskRuns are not getting deleted if the parent PipelineRun is running.
```

Comment:
@piyush-garg we need to inform users about the `--ignore-running-pipelinerun` right?

Reply:
Yes, that is added in features

Reply:
ok. :+1: